parser/prism/Translator.h

```diff
@@ -19,20 +19,26 @@ class Translator final {
     // but don't have explicit ownership over it. We take a temporary reference to it, but we can't
     // escape that scope, which is why Translator objects can't be copied, or even moved.
     core::GlobalState &gs;
+
+    // Context variables
     uint16_t uniqueCounter = 1;
+    bool isInMethodDef = false;
 
     Translator(Translator &&) = delete;                 // Move constructor
     Translator(const Translator &) = delete;            // Copy constructor
     Translator &operator=(Translator &&) = delete;      // Move assignment
     Translator &operator=(const Translator &) = delete; // Copy assignment
 public:
-    Translator(Parser parser, core::GlobalState &gs) : parser(parser), gs(gs) {} // Default constructor
```

Comment:
Hehe this was never the default constructor, since it had parameters. Such are the perils of comments.